src/eth/monitor.ts

```diff
@@ -35,7 +36,7 @@ async function withL2Client(cb: (l2Client: SubstrateClient) => Promise<void>) {
 async function handleCharge(v: DepositEventType, hash: string) {
   return withL2Client(async (l2Client: SubstrateClient) => {
     let l2account = toSS58(v.l2account);
-    console.log("Charge token_addr:", toHexStr(v.l1token));
+    addTimestampForLog("Charge token_addr: " + toHexStr(v.l1token));
```

Comment:
You had use addTimestampForLog(log: string, arg?: any), why we need change the input to single string?
Ideally we should not change the input for all places.